src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -155,7 +159,78 @@ def load_mlflow_model(
         _logger.info("Successfully installed model dependencies")
 
     try:
-        model = mlflow.pyfunc.load_model(model_uri)._model_impl
+        if not use_separate_conda_env:
+            model = mlflow.pyfunc.load_model(model_uri)._model_impl
+            return model
+
+        # Serve model from separate conda env using mlflow
+        mlflow_models_serve_logfile_name = "mlflow_models_serve.log"
+        try:
+            # run mlflow model server in background
+            with open(mlflow_models_serve_logfile_name, "w") as logfile:
+                model_serving_log = subprocess.Popen(
+                    [
+                        "mlflow",
```

Comment:
ideally these would be moved to some constants and re-used elsewhere, but since we are under time constraints I guess it can go in as-is